install_config/web_console_customization.adoc

```diff
@@ -172,6 +172,131 @@ assetConfig:
     - /path/to/cli-links.js
 ----
 ====
+
+[[adding-to-or-changing-navigation-menus]]
+=== Adding to or Changing Navigation Menus
+
+==== Top Navigation Dropdown Menus
+
+In the top navigation bar of the console there are two menu dropdowns, the help icon menu
+and the user menu.  Each of these menus can have additional menu items added to them.  They use
+the link:https://github.com/openshift/angular-extension-registry[angular-extension-registry].  A basic
+example of extending a menu is shown below, where 'myExtensionModule' is a placeholder and should be
+unique enough that it does not clash with any angular modules in the future.
+
+====
+----
+angular
+  .module('myExtensionModule', ['openshiftConsole'])
+  .run([
+    'extensionRegistry',
+    function(extensionRegistry) {
+      extensionRegistry
+        .add('nav-help-dropdown', function() {
+          return [
+            {
+              type: 'dom',
+              node: '<li><a href="http://www.example.com/report" target="_blank">Report a Bug</a></li>'
+            }, {
+              type: 'dom',
+              node: '<li class="divider"></li>'  // If you want a horizontal divider to appear in the menu
+            }, {
+              type: 'dom',
+              node: '<li><a href="http://www.example.com/status" target="_blank">System Status</a></li>'
+            }
+          ];
+        });
+    }
+  ]);
+
+hawtioPluginLoader.addModule('myExtensionModule');
+----
+====
+
+The available extension points are:
+
+* nav-help-dropdown - the help icon dropdown menu, visible at desktop screen widths
+* nav-user-dropdown - the user dropdown menu, visible at desktop screen widths
+* nav-dropdown-mobile - the single menu for top navigation items at mobile screen widths
+
+==== Project Left Navigation
+
+When you are navigating within a project, a menu appears on the left with primary and secondary navigation.
+This menu structure is defined as a constant and can be overridden or modified.
+
+[NOTE]
+====
+Significant customizations to the project navigation may affect the user experience and should be done with
+careful consideration. You will need to update this customization in future upgrades to guarantee you are picking
+up any new pages that are available.
```

Comment:
If we ask them to replace the entire object, we'd need to give them some link to constants.js with the menu for the version of OpenShift they have. The approach above would avoid that.